client/src/components/ModelViewerComponent/ModelViewerComponent.test.js

```diff
@@ -0,0 +1,307 @@
+import React from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+import { act, Simulate } from "react-dom/test-utils";
+import { findByText, findByLabelText, waitFor, findByTestId, fireEvent, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import PubSub from "pubsub-js";
+import ModelViewerComponent from "./ModelViewerComponent";
+import { apiService } from "../../services/ApiService";
+import { configService } from "../../services/ConfigService";
+import { eventService } from "../../services/EventService";
+import { settingsService } from "../../services/SettingsService";
+import { ModelService } from "../../services/ModelService";
+import { ConsoleComponent } from "../ConsoleComponent/ConsoleComponent";
+import initIcons from "../../services/IconService/IconService";
+
+initIcons();
+
+
+jest.mock("../../services/ApiService");
+jest.mock("../../services/ConfigService");
+jest.mock("../../services/ModelService");
+jest.mock("../../services/EventService");
+jest.mock("../../services/SettingsService");
+
+const retrieveModels = jest.spyOn(apiService, "queryModels");
+const deleteModel = jest.spyOn(apiService, "deleteModel");
+const uploadModel = jest.spyOn(apiService, "addModels");
+const getModel = jest.spyOn(apiService, "getModelById");
+const getModelImage = jest.spyOn(settingsService, "getModelImage");
+const deleteModelImage = jest.spyOn(settingsService, "deleteModelImage");
+
+const models = [
+  {
+    "displayName": {
+      "en": "Floor"
+    },
+    "description": {},
+    "id": "dtmi:com:example:adtexplorer:Floor;1",
+    "uploadTime": "2022-02-10T19:57:28.372Z",
+    "decommissioned": false,
+    "model": {
+      "@id": "dtmi:com:example:adtexplorer:Floor;1",
+      "@type": "Interface",
+      "@context": [
+        "dtmi:dtdl:context;2"
+      ],
+      "displayName": "Floor",
+      "contents": [
+        {
+            "@type": "Relationship",
+            "name": "contains",
+            "target": "dtmi:com:example:adtexplorer:Room;1"
+        },
+        {
+            "@type": "Property",
+            "name": "AverageTemperature",
+            "schema": "double",
+            "writable": true
+        }
+      ]
+    }
+  }
+];
+
+const mockGetModelImage = "\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARwAAACxCAMAAAAh3/JWAAABiVBMVEWAgIAAAAD////AwAB9fX2epllnWahYpo6mWXkAwL++AAIAvwC/v796enpajqalcVm/AL8AAb6wsLCZmZlmZmaNjY1zc3Ourq7MzMwzMzOkpKSFhYWRkZFfX1+goKCPjwBWVlZkVapHR0eTkZmYmZHd3d0NDQ0dHR3a2tp8r566vpvJycmvfZF7gH6hm79QUFAqKiphpZuEaZePo30AwMju7u6raHdSB5+oclOjsbiGm6e3qaOfk46kvksIvV68CJQBz86njoTNAQPX13/PzwEADxABzgHOAc4PDwB6eogAABABA82YoUsEnJt0uHS4dLiBk42RjqBjwL8EmwRkwGSbBpu/Y7+/Y2OaBAWdnQRgYL+Ghnq+vmAGBpufn1m2tieQkG3Q0JSlpUOYmGOwsDu7uxiZrrm3w8ofBAVeV...";
+
+const mockSuccesResponse = { "Status": "Success"};
```

Comment:
Update spelling as per above comment.

Reply:
Has this been fixed?

Also, add a space between the value and the closing bracket.